source/common/grpc/common.cc

```diff
@@ -53,11 +54,12 @@ void Common::chargeStat(const Upstream::ClusterInfo& cluster, const std::string&
   }
   cluster.statsScope()
       .counter(fmt::format("{}.{}.{}.{}", protocol, grpc_service, grpc_method,
-                           grpc_status->value().c_str()))
+                           grpc_status->value().getStringView()))
       .inc();
   uint64_t grpc_status_code;
+  std::string grpc_status_string(grpc_status->value().getStringView());
```

Comment:
nit: const here and elsewhere.

Reply:
Done, though my eyes glaze over reading this diff so please let me know if I missed another :)